api/apistruct/types.go

```diff
@@ -0,0 +1,12 @@
+package apistruct
+
+import "github.com/filecoin-project/lotus/api"
+
+type ChainIO = api.ChainIO
+type Common = api.Common
+type FullNode = api.FullNode
+type Gateway = api.Gateway
+type Signable = api.Signable
+type StorageMiner = api.StorageMiner
+type Wallet = api.Wallet
+type Worker = api.Worker
```

Comment:
This is here only for the generated type checks at the bottom of struct.go

I'm not sure why `apistruct` is a separate package, maybe it's worth merging it with `api`